hazelcast/src/test/java/com/hazelcast/internal/serialization/DataInputOutputTest.java

```diff
@@ -43,7 +39,7 @@
 @Category(QuickTest.class)
 public class DataInputOutputTest {
 
-    private final Person person = new Person(111, 123L, 89.56d, "test-person", new Address("street", 987));
+    private final SerializationConcurrencyTest.Person person = new SerializationConcurrencyTest.Person(111, 123L, 89.56d, "test-person", new SerializationConcurrencyTest.Address("street", 987));
```

Comment:
You lost the static imports of `Address` and `Person` here (seems to be an issue with IntelliJ and moving files).